### PR TITLE
Dx 686 proper error message

### PR DESCRIPTION
**Pull request by sarveshdream11**

### CODE REVIEW

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -81,3 +82,7 @@ func FetchKey(keyName string) string {
 	f := reflect.Indirect(r).FieldByName(keyName)
 	return f.String()
 }
+
+func GetUUID() string {
```
Comment (BalkishanNanavath) [inline pkg/utils/utils.go:86]: why is this required?

Comment (sarveshdream11) [reply pkg/utils/utils.go:86]: removed

**Review by BalkishanNanavath — APPROVED** [review]

File: internal/constant/const.go [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package constant
```
Comment (dhavalmehta07) [inline internal/constant/const.go:1]: Should we call this errors, so that only and all error descriptions go in here?

Comment (sarveshdream11) [reply internal/constant/const.go]: done

**Review by dhavalmehta07 — APPROVED** [review]
